### DISCUSSION TITLE
How to get active item index on Carousel

### DISCUSSION THREAD

**Discussion by bezchristo** [saadeghi/daisyui#628] [Q&A]
I am trying to use the [Carousel](https://daisyui.com/components/carousel) for a personal project but I am running into a bit of a hurdle. I want to display a few dots underneath the carousel to indicate which item is visible. In order to do that I need to be able to get the current visible item index somehow. Is there a way for me to do that? I can set the item when I click on a dot, but how will I get it when I scroll with my finger through the carousel?

![image](https://user-images.githubusercontent.com/1775663/158967482-1ca5ebe7-820d-4055-84ef-a5514a9ac962.png)

**Comment by saadeghi** [comment]
I'm afraid that's not possible without JS. daisyUI's carousel is just a scrollable area with scroll snap.  We can have bullets or next/prev buttons using HTML anchor links but that's where a CSS-only image slider has limits, any other functionalities should be handled using JavaScript.
